status-loop/getChannels.js

```diff
@@ -0,0 +1,27 @@
+const getRequest = require('../helpers/getRequest')
+const cfg = require('../cfg')
+
+async function getChannels () {
+	// const lists = хawait Promise.all(cfg.initialValidators.map(url => getRequest(`${url}/channel/list`)))
+	let { channels } = await getRequest(`${cfg.initialValidators[0]}/channel/list`)
+	// Ensuring it would work if we change total to totalPages
+	if ((channels.total && channels.total > 1) || (channels.totalPages && channels.totalPages > 1)) {
```

Comment:
add a comment here that this should be fixed

Reply:
You can remove `channels.total` now, it's been changed to `totalPages` on the JS validator stack implementation